src/CMakeLists.txt

```diff
@@ -91,6 +109,11 @@ if (BUILD_TESTING)
     -DGZ_SDFORMAT_STATIC_DEFINE
   )
 
+  if(WIN32)
+    target_link_libraries(${PROJECT_LIBRARY_TARGET_NAME}
+      PRIVATE shlwapi)
```

Comment:
maybe this needs to be public?

Reply:
actually this `target_link_libraries` call is inside `if (BUILD_TESTING)`, but it should no depend on the value of `BUILD_TESTING`. I'll submit a PR shortly

Reply:
fix in https://github.com/gazebosim/sdformat/pull/1468